docs/throughs/take.md

```diff
@@ -1,13 +1,55 @@
 # pull-stream/throughs/take
 
-## usage
+## Example usage
 
-### `take = require('pull-stream/throughs/take')`
+```js
+var pull = require('pull-stream')
+// var take = require('pull-stream/throughs/take') // if you just need take
```

Comment:
Does this line need to be commented out?

Reply:
yep, it's not used. I'm showing an alternative.

This PR is of the class of "it's better than it was", so I'm not going to tune this particular thing. Thanks for input though :)